pybamm/input/parameters/lithium_ion/Marquis2019.py

```diff
@@ -380,6 +380,7 @@ def get_parameter_values():
         "Inner SEI reaction proportion": 0.5,
         "Inner SEI partial molar volume [m3.mol-1]": 9.585e-05,
         "Outer SEI partial molar volume [m3.mol-1]": 9.585e-05,
+        "SEI growth transfer coefficient": 0.5,
```

Comment:
I am not sure why this is not working.

Reply:
We might need to do this to all the parameter sets (which might not be a great idea). Maybe it is better to either add it in the benchmarks directly or skip the asymmetric SEI models.

Reply:
The failing asymmetric benchmarks are only running with "Marquis2019", are they using the other parameter values internally?